libs/application/templates/family-matters/children-residence-change/src/lib/ChildrenResidenceChangeTemplate.ts

```diff
@@ -158,11 +160,15 @@ const ChildrenResidenceChangeTemplate: ApplicationTemplate<
           ],
         },
       },
-      [ApplicationStates.REJECTED]: {
+      [ApplicationStates.REJECTEDBYPARENTB]: {
         meta: {
           name: applicationName,
-          progress: 1,
+          description: stateDescriptions.rejectedByParentB,
```

Comment:
To follow the same example as the lifecycle object, I would use an object to define theses fields. We are already in the meta object, so not sure what the name should be but something like

```
obj(state?,info?): {
  title: '',
  description: '',
  tag: {
    variant: '',
    label: '',
  }
}
```